arduino_sketch/arduino_sketch.ino

```diff
@@ -1,67 +1,30 @@
 #include <Arduino.h>
-#include <DHT.h>        // For the DHT11 temperature and humidity sensor
-#include <Time.h>       // For timekeeping
-#include <Wire.h>       // For I2C
+#include <TimeLib.h>  // For timekeeping
 
-// Define LED pins (PWM pins: 3,5,6,9,10,11 on arduino UNO)
-const int RED   = 11;
-const int GREEN = 10;
-const int BLUE  = 9;
-const int PUMP  = 2;
-const int NOZZLE = 12;
-const int FAN = 7;
-const int BOARDLED = 13; // Pin 13 and Arduino UNO board LED
-const int LIGHT = A0;    // Light sensor, Analog pin 0
+// LED pins (PWM pins: 3,5,6,9,10,11 on arduino UNO)
+const int PUMP  = 4;  // Pin 4 on arduino maps to D2 on wemos
+const int NOZZLE = 5; // Pin 5 on arduino maps to D1 on wemos
 const int runInterval = 1;
-const int measureInterval = 3600;
-
-
-// Define DHT11 sensor
-#define DHTPIN 8          // What pin we're connected to
-#define DHTTYPE DHT11     // DHT 11
-DHT dht(DHTPIN, DHTTYPE); // Make the object
+const int pumpMaxRunTime = 9;
```

Comment:
Have this at 0 to run the pump for one min is enough for most setups. Increase this number if you have a larger pressure tank. (Something like 40L or more).